package.json

```diff
@@ -55,15 +55,15 @@
     "expect": "^1.20.2",
     "got": "^9.6.0",
     "http-proxy-middleware": "0.19.1",
-    "mocha": "^6.1.4",
-    "node-sass": "^4.12.0",
-    "nodemailer": "^6.2.1",
-    "nodemon": "^1.19.1",
+    "mocha": "^6.2.2",
+    "node-sass": "^4.13.0",
+    "nodemailer": "^6.4.2",
+    "nodemon": "^1.19.4",
     "nyc": "^14.1.1",
     "standard": "^12.0.1"
   },
   "engines": {
-    "node": ">=8.0.0"
+    "node": ">=12.0.0"
```

Comment:
> I could change this to V10(Active LTS until 2021) if the version is too high?
Yes, @jovanmaric, can you change this to 10? With 10 in LTS, I don't want users running 10 to get an error w/ this new min engine requirement unless there is another core dependency in 12 that maildev requires. Thanks!